tests/Message/PurchaseRequestTest.php

```diff
@@ -97,7 +103,7 @@ public function testGetRegisterPayData()
         $this->assertSame('http://cancel', $data['vads_url_cancel']);
         $this->assertSame('http://error', $data['vads_url_error']);
         $this->assertSame('http://refused', $data['vads_url_refused']);
-        $this->assertSame('2a83b1bbd0cd09e01a7e1c8a0beaacff73fa75dd', $data['signature']);
+        $this->assertSame('4cbe6302811fcf57235c5f6b60b33c153e4bb852', $data['signature']);
         $this->assertSame('a owner reference', $data['vads_ext_info_owner_reference']);
         $this->assertSame('1', $data['vads_ext_info_info1']);
         $this->assertSame('2', $data['vads_ext_info_info2']);
```

Comment:
We can also check here if the `vads_redirect_success_timeout` and `vads_redirect_error_timeout` are defined